src/_pytest/capture.py

```diff
@@ -1004,6 +1004,40 @@ def test_output(capsys):
     capman.unset_fixture()
 
 
+@fixture
+def capteesys(request: SubRequest) -> Generator[CaptureFixture[str]]:
```

Comment:
this fixture is almost 100% a copy of the `capsys` one above it, since a `tee-sys` is just a `sys` w/ an extra argument (`tree=True`)